Thanks @dmitrytokarev. It looks like the unit test for kind_sorter_test.go failed. Would you please look into that?

```
--- FAIL: TestKindSorter (0.00s)
    --- FAIL: TestKindSorter/uninstall (0.00s)
        kind_sorter_test.go:182: Expected "wvmutsxrqponLlKkJjIi2hgf1ed3CcbAa!", got "wvmutsxrqponLlKkJjIi2hgf1edcba!A3C"
```

--- a/pkg/releaseutil/kind_sorter.go
+++ b/pkg/releaseutil/kind_sorter.go
@@ -26,8 +26,11 @@ type KindSortOrder []string
 // Those occurring earlier in the list get installed before those occurring later in the list.
 var InstallOrder KindSortOrder = []string{
 	"Namespace",
+	"NetworkPolicy",
 	"ResourceQuota",
 	"LimitRange",
+	"PodSecurityPolicy",
+	"PodDisruptionBudget",
 	"Secret",
 	"ConfigMap",
 	"StorageClass",
@@ -88,8 +91,11 @@ var UninstallOrder KindSortOrder = []string{
 	"StorageClass",
 	"ConfigMap",
 	"Secret",
+	"PodDisruptionBudget",
+	"PodSecurityPolicy",
 	"LimitRange",
 	"ResourceQuota",
+	"NetworkPolicy",
 	"Namespace",
 }
 

--- a/pkg/releaseutil/kind_sorter_test.go
+++ b/pkg/releaseutil/kind_sorter_test.go
@@ -80,6 +80,10 @@ func TestKindSorter(t *testing.T) {
 			Head: &SimpleHead{Kind: "Namespace"},
 		},
 		{
+			Name: "A",
+			Head: &SimpleHead{Kind: "NetworkPolicy"},
+		},
+		{
 			Name: "f",
 			Head: &SimpleHead{Kind: "PersistentVolume"},
 		},
@@ -90,6 +94,14 @@ func TestKindSorter(t *testing.T) {
 		{
 			Name: "o",
 			Head: &SimpleHead{Kind: "Pod"},
+		},
+		{
+			Name: "3",
+			Head: &SimpleHead{Kind: "PodDisruptionBudget"},
+		},
+		{
+			Name: "C",
+			Head: &SimpleHead{Kind: "PodSecurityPolicy"},
 		},
 		{
 			Name: "q",
@@ -154,8 +166,8 @@ func TestKindSorter(t *testing.T) {
 		order       KindSortOrder
 		expected    string
 	}{
-		{"install", InstallOrder, "abcde1fgh2iIjJkKlLmnopqrxstuvw!"},
-		{"uninstall", UninstallOrder, "wvmutsxrqponLlKkJjIi2hgf1edcba!"},
+		{"install", InstallOrder, "aAbcC3de1fgh2iIjJkKlLmnopqrxstuvw!"},
+		{"uninstall", UninstallOrder, "wvmutsxrqponLlKkJjIi2hgf1ed3CcbAa!"},
 	} {
 		var buf bytes.Buffer
 		t.Run(test.description, func(t *testing.T) {

